index.html

```diff
@@ -1210,6 +1300,16 @@ <h3>
             Updating the manifest
           </h3>
           <aside class="issue" data-number="446"></aside>
+          <p>
+            User agents MAY, at any time, re-fetch and update a manifest for an
```

Comment:
Thinking about this some more, the manifest should be re-fetched on every page load, like any other linked resource. If changes are detected, then the manifest should be re-applied and significant changes flagged to the end-user.

We need to better handle what happens when the ID changes. Also, we need to handle what happens if a manifest is in use, but the new manifest is parsed and it fails.

Reply:
Chromium currently fetches on every page load. See my comment above about when the changes apply.

Different ID means that this page is referencing a different webapp, and if user agents choose to they can advertise a different install.

Failed parsing means, I believe, we can't do anything. We don't know if it would apply to a current install or a new install